src/Alerts/components/IssuedAlerts/Filters/index.js

```diff
@@ -1,113 +1,115 @@
 import {
-  clearAlertFilters,
+  clearFocalPersonFilters,
```

Comment:
@Mloweedgar Hello Bro .. I think it was supposed to be filters

Reply:
Hello bro @BenMaruchu  that is fixed in PR #19. once you merge #19 everything will be fine